ops/update.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/bash
 
 git fetch
+poetry run python dbmate.py up
```

Comment:
We should not run this here. This script runs on a schedule, and spends a lot of time doing nothing. So, any actual updates should happen inside the `if` below, after we actually pull the changes below, before we tell systemd to reload the daemon.

Reply:
ah, ill document and fix that then, i had assumed the if contained stuff if the docker container had changed

Reply:
The if logic is basically "does the remote branch's commit hash not match the local branch's hash"